public/sketch.js

```diff
@@ -135,12 +145,12 @@ function draw() {
   background(255);
   // noStroke();
   
-  var freq = 0;
+  //var freq = 0;
```

Comment:
The beautiful thing about version control is you can safely delete these things and not worry about losing them -- i.e. they're saved in previous versions of the codebase!

Reply:
So in general, you don't want commented code in your pull requests.